ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/MarkdownView.kt

```diff
@@ -1,73 +1,104 @@
 package org.jetbrains.kotlinconf.ui.components
 
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.platform.UriHandler
+import androidx.compose.ui.text.SpanStyle
 import androidx.compose.ui.text.TextLinkStyles
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.unit.dp
 import com.mikepenz.markdown.compose.Markdown
 import com.mikepenz.markdown.model.DefaultMarkdownColors
 import com.mikepenz.markdown.model.DefaultMarkdownTypography
+import com.mikepenz.markdown.model.ReferenceLinkHandler
+import com.mikepenz.markdown.model.ReferenceLinkHandlerImpl
 import com.mikepenz.markdown.model.markdownAnimations
+import com.mikepenz.markdown.model.markdownAnnotator
+import com.mikepenz.markdown.model.markdownAnnotatorConfig
 import com.mikepenz.markdown.model.markdownPadding
+import com.mikepenz.markdown.model.rememberMarkdownState
 import org.jetbrains.kotlinconf.ui.theme.KotlinConfTheme
 
 @Composable
 fun MarkdownView(
     loadText: suspend () -> ByteArray,
     modifier: Modifier = Modifier,
+    onUriClick: (String) -> Unit = {},
 ) {
     var text by remember { mutableStateOf("") }
     LaunchedEffect(loadText) {
         text = loadText().decodeToString()
     }
 
-    MarkdownView(text, modifier)
+    MarkdownView(text, modifier, onUriClick)
 }
 
 @Composable
 fun MarkdownView(
     text: String,
     modifier: Modifier = Modifier,
+    onUriClick: (String) -> Unit = {},
```

Comment:
The name is a bit misleading, I was about to write a comment, why no URIs are handled by default.
Maybe rename it to something like `onCustomUriClick`, `onInAppUriClick`, or `onLocalUriClick`?